tests/common/widgets/breadcrumb/options.ts

```diff
@@ -296,5 +291,169 @@ export function createOptionsTests(setup: BreadcrumbSetup, act: Act) {
         ])
       );
     });
+
+    test('specifies a custom separator in records', async () => {
+      const hierarchicalFacetsWithCustomSeparator = {
+        'hierarchicalCategories.lvl0': {
+          'Cameras & Camcorders': 1369,
+        },
+        'hierarchicalCategories.lvl1': {
+          'Cameras & Camcorders ~ Digital Cameras': 170,
+        },
+      };
+
+      const searchClient = createMockedSearchClient({
+        facets: hierarchicalFacetsWithCustomSeparator,
+      });
+
+      await setup({
+        instantSearchOptions: {
+          indexName: 'indexName',
+          initialUiState: {
+            indexName: {
+              hierarchicalMenu: {
+                'hierarchicalCategories.lvl0': [
+                  'Cameras & Camcorders ~ Digital Cameras',
+                ],
+              },
+            },
+          },
+          searchClient,
+        },
+        widgetParams: {
+          attributes: Object.keys(hierarchicalFacetsWithCustomSeparator),
+          separator: ' ~ ',
+        },
+      });
+
+      await act(async () => {
+        await wait(0);
+      });
+
+      expect(
+        document.querySelector('.ais-Breadcrumb')
+      ).toMatchNormalizedInlineSnapshot(
+        normalizeSnapshot,
+        `
+        <div
+          class="ais-Breadcrumb"
+        >
+          <ul
+            class="ais-Breadcrumb-list"
+          >
+            <li
+              class="ais-Breadcrumb-item"
+            >
+              <a
+                class="ais-Breadcrumb-link"
+                href="#"
+              >
+                Home
+              </a>
+            </li>
+            <li
+              class="ais-Breadcrumb-item"
+            >
+              <span
+                aria-hidden="true"
+                class="ais-Breadcrumb-separator"
+              >
+                &gt;
+              </span>
+              <a
+                class="ais-Breadcrumb-link"
+                href="#"
+              >
+                Cameras & Camcorders
+              </a>
+            </li>
+            <li
+              class="ais-Breadcrumb-item ais-Breadcrumb-item--selected"
+            >
+              <span
+                aria-hidden="true"
+                class="ais-Breadcrumb-separator"
+              >
+                &gt;
```

Comment:
maybe an extra assertion explicitly on the `&gt;` could be useful, as the separator in the widget isn't taken in account for the rendering

Reply:
Good idea, addressed in 0bb5937bb.